wake-lock/idlharness.https.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Wake Lock API IDL tests</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/wake-lock/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/idlharness.js"></script>
+<script>
+"use strict";
+
+function doTest([dom_idl, wakelock_idl, wakelock]) {
+  const idl_array = new IdlArray();
+  idl_array.add_untested_idls(dom_idl);
```

Comment:
We should actually test this too. They depend on each other, so why not just test them all?

Reply:
The main DOM IDL has been tested in https://github.com/w3c/web-platform-tests/blob/master/dom/interfaces.html, this would avoid duplication.

Reply:
It depends: I'm ok to not use it, so long as there is no dependency on each other. 

I can't recall if there is or not - but if this spec depends on anything in DOM being there, we should definitely test it.